src/resources/AmalgamationRegular/BC.ts

```diff
@@ -29,9 +29,6 @@ export const AmalgamationRegResourceBc: AmalgamationResourceIF = {
       }
     ]
   },
-  shareClasses: {
-    countMinimum: 1
-  },
```

Comment:
This is not used. It looks like [ShareStructure.vue](https://github.com/bcgov/business-create-ui/blob/4a874e2d48031d395c918783bab353e820cfbee9/src/views/Amalgamation/ShareStructure.vue#L86) directly checks for at least 1 share instead of using resources.